### ISSUE TITLE
Reviewer suggestions

### ISSUE THREAD

**Issue by Benjamin-Lee** [Benjamin-Lee/deep-rules#227]
_PLOS Computational Biology_ requires us to propose four reviewers. Does anyone have suggestions for who would could propose?

**Comment by rasbt** [comment]
Here would be two. Before submitting, we need to make sure none of the co-authors is affiliated / friends with any of the reviewers we suggest.

- Enrico Ferrero, https://scholar.google.com/citations?user=HAl-8xUAAAAJ&hl=en, enrico.ferrero@novartis.com
- Christof Angermueller, https://scholar.google.com/citations?user=OXZC0mQAAAAJ, admin@cangermueller.com

**Comment by cgreene** [comment]
* Su-In Lee https://scholar.google.com/citations?user=3ifikJ0AAAAJ&hl=en
* Joel Dudley https://scholar.google.com/citations?user=206DEM0AAAAJ&hl=en

Looking at our author list, between the people and institutions that's a lot of potential COIs on folks who I would otherwise suggest. We don't have any folks from the UC system, and there are also a lot of potential reviewers there.

**Comment by Benjamin-Lee** [comment]
@cgreene and @rasbt, thank you for your suggestions. Here are the PLOS guidelines for COIs:

- They work at the same institution or organization as an author, currently or recently
- They collaborate with an author, currently or recently
- They have published with an author during the past 5 years
- They have held grants with an author, currently or recently
- They have a financial relationship with the company who funded the research
- They have a personal relationship with an author that does not allow them to evaluate the manuscript objectively

**Comment by rasbt** [comment]
Thanks for posting. Unless there are further suggestion, I suggest we keep the 4 researchers above as tentative reviewers to suggest. Everyone included in the author list would need to confirm that there is no COI. Maybe we should be tagging people here to make sure this gets seen before submission

**Comment by Benjamin-Lee** [comment]
@rasbt, good idea. Let's lock those in. I'll add checking for conflicts to #226. The only possible issue is that Enrico Ferrero is a coauthor on Deep Review, so he has conflicts with many of the current coauthors.

**Comment by cgreene** [comment]
This is a set of suggested reviewers so if we end up accidentally listing one with a conflict, that person will just decline to review. If we're going to have action items for everyone, I think focusing on the things like manuscript review, revision, and approval is the best area to put attention and eyes. Does the PLOS CB system allow you to list _more_ than 4 reviewers? If so, we could add to this list to make it as easy as possible for the editor to identify reviewers. If some had conflicts with an author here and declined, it wouldn't make it much harder for the editor to find folks.

**Comment by agitter** [comment]
A few more suggestions.  I haven't cross-referenced potential conflicts:
- James Zou https://scholar.google.com/citations?user=23ZXZvEAAAAJ&hl=en&oi=ao
- Anna Goldenberg https://scholar.google.com/citations?user=cEepZOEAAAAJ&hl=en&oi=ao
- Smita Krishnaswamy https://scholar.google.com/citations?user=l2Pr9m8AAAAJ&hl=en&oi=ao

**Comment by juancarmona** [comment]
I strongly recommend **Dr. Helen van Aggelen**. I previously collaborated and published with Helen when she was employed at our Philips office in Cambridge, MA. She is now based in California at a company called Zymergen. Helen's life sciences output spans an impressive array of pre-clinical research and clinical studies involving AI/ML. There is no conflict of interest on my end regarding this nomination.

- https://orcid.org/0000-0002-4934-0300
- https://scholar.google.com/citations?user=8tEc3jYAAAAJ&hl=en&oi=sra
